opentelemetry-api/src/opentelemetry/metrics/examples/pre_aggregated.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from opentelemetry.metrics import Meter
+from opentelemetry.metrics.label_key import LabelKey
+
+METER = Meter()
+LABEL_KEYS = [LabelKey("environment", 
+                       "the environment the application is running in")]
+SUM_METRIC = METER.create_int_counter("sum numbers", 
+                                      "sum numbers over time",
+                                      "number",
+                                      LABEL_KEYS)
+LABEL_VALUES = ["Testing"]
+SUM_TIME_SERIES = SUM_METRIC.getOrCreateTimeSeries(LABEL_VALUES)
```

Comment:
Thanks for adding the example! Things are getting a little clearer for me.

What is the value for having the workflow of creating a metric, then creating a time series?

It feels to me like the metric is not valuable without the time series included. Generally, when emitting metrics, I think of a single measurement as an event that writes to a stream, that then gets aggregated by whatever queue system and submitted to the exporter.

If I was a user, I would like things to be as simple as:

```
METER = Meter()
LABEL_KEYS = [LabelKey("environment", 
                       "the environment the application is running in")]
SUM_METRIC = METER.create_int_counter("sum numbers", 
                                      "sum numbers over time",
                                      "number",
                                      LABEL_KEYS)
SUM_METRIC.record(10)
```